samples/04_webhook_client/Program.cs

```diff
@@ -0,0 +1,34 @@
+using Discord;
+using Discord.Webhook;
+using System.Threading.Tasks;
+
+namespace _04_webhook_client
+{
+    // This is a minimal example of using Discord.Net's Webhook Client
+    // Webhooks are send-only components of Discord that allow you to make a POST request
+    // To a channel specific URL to send a message to that channel.
+    class Program
+    {
+        static void Main(string[] args)
+            => new Program().MainAsync().GetAwaiter().GetResult();
+
+        public async Task MainAsync()
+        {
+            // The webhook url follows the format https://discordapp.com/api/webhooks/{id}/{token}
+            // Because anyone with the webhook URL can use your webhook
+            // you should NOT hard code the URL or ID + token into your application.
+            using (var client = new DiscordWebhookClient(webhookId: 0, webhookToken: ""))
```

Comment:
```suggestion
            using (var client = new DiscordWebhookClient(Environment.GetEnvironmentVariable("WEBHOOK_URL")))
```
We can update this to use the new constructor. Comments should be updated to say that the env var needs to be set, or we can just let this example have a hardcoded string (since that's what it already was, effectively)

Reply:
Enough people get tripped up over the use of environment variables in the samples, I opted to just throw a hardcoded dummy url into it instead.